Rakefile

```diff
@@ -15,7 +15,7 @@ SAMPLE_IOS = "Project"
 
 
 # Destinations
-IPHONE_DEST = (ENV["IPHONE_DEST"] == nil || ENV["IPHONE_DEST"] == "null") ? 'platform=iOS Simulator,OS=latest,name=iPhone 5' : ENV["IPHONE_DEST"]
+IPHONE_DEST = (ENV["IPHONE_DEST"] == nil || ENV["IPHONE_DEST"] == "null") ? 'platform=iOS Simulator,OS=latest,name=iPhone SE' : ENV["IPHONE_DEST"]
```

Comment:
Out of curiosity what's the reason for this?

Reply:
RA tests were not running with an error complaining about the simulator iPhone version.  I'll test again to confirm.

Reply:
Here's the `rake testios` output:
```
[CDT_test-IAM-jenkins-ORSF7CTM2EENSTMYD5LWS2L2LFXKBP4FAEOLZSTOZQSMP5P6LKNQ] Running shell script
+ rake testios
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
		{ platform:iOS Simulator, OS:latest, name:iPhone 5 }

	The requested device could not be found because no available devices matched the request.

	Available destinations for the "CDTDatastoreTests" scheme:
		{ platform:iOS, id:c98903079a8fde8a181f1fcfd84b9e4ed84b7d50, name:iPod }
		{ platform:iOS Simulator, id:817CE432-E2BD-47F7-BC03-65D95C9577E7, OS:11.0, name:iPad (5th generation) }
		{ platform:iOS Simulator, id:18E7B059-95B7-40D1-BF91-CE018CA4F8F8, OS:11.0, name:iPad Air }
		{ platform:iOS Simulator, id:0EF5B18A-E22A-4D75-A61B-121E494D09C4, OS:11.0, name:iPad Air 2 }
		{ platform:iOS Simulator, id:285D53A6-F479-41F2-878B-A96D3A792737, OS:11.0, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:2FE186DD-617F-4342-AFE1-3EC28810B633, OS:11.0, name:iPad Pro (10.5-inch) }
		{ platform:iOS Simulator, id:CFC00305-150F-44A7-A391-447DAEE9657E, OS:11.0, name:iPad Pro (12.9-inch) }
		{ platform:iOS Simulator, id:FE89E927-58EE-4B82-9B54-5A299D30D179, OS:11.0, name:iPad Pro (12.9-inch) (2nd generation) }
		{ platform:iOS Simulator, id:580B93D6-2C22-47A1-9BC8-026CD6FF6E4D, OS:11.0, name:iPhone 5s }
		{ platform:iOS Simulator, id:B67B75EB-AA50-444C-BA28-53AE4B642CFD, OS:11.0, name:iPhone 6 }
		{ platform:iOS Simulator, id:1760030C-257A-4C5D-AF05-8CF1FFC8F6DB, OS:11.0, name:iPhone 6 Plus }
		{ platform:iOS Simulator, id:45A6EF6C-A522-4901-8FED-8BD483CADFA6, OS:11.0, name:iPhone 6s }
		{ platform:iOS Simulator, id:08A25593-346B-4A30-BD94-7903E2A39FF5, OS:11.0, name:iPhone 6s Plus }
		{ platform:iOS Simulator, id:23D65A6D-DA7B-488A-B709-126660058D01, OS:11.0, name:iPhone 7 }
		{ platform:iOS Simulator, id:6D3C313C-C4FE-4850-81A5-2992E57828CC, OS:11.0, name:iPhone 7 Plus }
		{ platform:iOS Simulator, id:52A6F190-4C24-4D64-AE75-E8C40FF7CF60, OS:11.0, name:iPhone 8 }
		{ platform:iOS Simulator, id:1E595F43-F52F-45A1-8D78-26104A91917B, OS:11.0, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:4D50B7E8-8821-48E4-9332-B1AAD6F28C3F, OS:11.0, name:iPhone SE }
		{ platform:iOS Simulator, id:1B679759-0684-40F9-A017-F0480EAFD940, OS:11.0, name:iPhone X }

	Ineligible destinations for the "CDTDatastoreTests" scheme:
		{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Generic iOS Device }
		{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Generic iOS Simulator Device }
rake aborted!
[FAILED] Tests CDTDatastore.xcworkspace, CDTDatastoreTests
/Users/cloudantsync/jenkins-ios/workspace/CDT_test-IAM-jenkins-ORSF7CTM2EENSTMYD5LWS2L2LFXKBP4FAEOLZSTOZQSMP5P6LKNQ/Rakefile:117:in `test'
/Users/cloudantsync/jenkins-ios/workspace/CDT_test-IAM-jenkins-ORSF7CTM2EENSTMYD5LWS2L2LFXKBP4FAEOLZSTOZQSMP5P6LKNQ/Rakefile:67:in `block in <top (required)>'
Tasks: TOP => testios
(See full trace by running task with --trace)
script returned exit code 1
```